angular/core/components/group_page/membership_request_form/membership_request_form.haml

```diff
@@ -7,11 +8,11 @@
     %fieldset
       .lmo-form-group
         %label{for: "membership-request-name", translate: "membership_request_form.name_label"}
-        %input.membership-request-form__name.form-control#membership-request-name{ng-model: "membershipRequest.actor().name", ng-required: "true", ng-disabled: "true"}
```

Comment:
Instead of relying on an actor, we'll initialize our membership request with the current user's name and email.

Then, the request name / email is only editable if you're not logged in.